cpu/stm32/include/clk/f0f1f3/cfg_clock_default.h

```diff
@@ -26,6 +26,10 @@
 #ifndef CLK_F0F1F3_CFG_CLOCK_DEFAULT_H
 #define CLK_F0F1F3_CFG_CLOCK_DEFAULT_H
 
+#include "cfg_clock_common_fx_gx_mp1_c0.h"
+#include "kernel_defines.h"
+#include "macros/units.h"
```

Comment:
```suggestion
#include "kernel_defines.h"
#include "macros/units.h"
#include "cfg_clock_common_fx_gx_mp1_c0.h"
```

That should fix [Murdock](https://ci.riot-os.org/results/fed706cc33a7419d8594223a9b414f1c/output/builds/examples/gnrc_networking_subnets/bluepill-stm32f030c8:gnu.txt)

Reply:
That's a bit reliant on no-obvious behavior for my tastes. I just explicitly added the missing includes to the files that needed them (in a fixup commit). If that is acceptable, let me know.

Reply:
That's good. Please squash

Reply:
done